Bill3.java

```diff
@@ -1,8 +1,14 @@
 // Splits a restaurant bill evenly among three diners.
 public class Bill3 {
 	public static void main(String[] args) {
+		
```

Comment:
Generally you could have written it much better.
int total = Integer.parseInt(args[3]); 
... 
double splittedAmount = Math.ceil( (double) (total / 3) );
...